contracts/PrecedenceCampaignArbitrable.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^0.5.8;
+
+import "@aragon/court/contracts/arbitration/IArbitrable.sol";
+import "@aragon/court/contracts/arbitration/IArbitrator.sol";
+
+
+contract PrecedenceCampaignArbitrable is IArbitrable {
+    //bytes4 public constant ERC165_INTERFACE = ERC165_INTERFACE_ID;
+    //bytes4 public constant ARBITRABLE_INTERFACE = ARBITRABLE_INTERFACE_ID;
+    string public constant ERROR_NOT_ALLOWED = "PCA_NOT_ALLOWED";
+
+    address public owner;
```

Comment:
Maybe we can rename this `disputeManager`?

Reply:
To me this sounds confusing with Aragon Court DisputeManager, maybe I'm biased.

Reply:
Right, that makes sense.